File: OCH_Tao/ABC/A/295.js
const W = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[1].split(" ");
const X = ["and", "not", "that", "the", "you"];
for (let i = 0; i < 5; i++) {
  if (W.includes(X[i])) {
    console.log("Yes");
    break;
  }
  if (i === 4) {
    console.log("No");
  }
}

File: OCH_Tao/ABC/A/295.py
N = int(input())
W = input().split()
if "and" in W:
  print("Yes")
elif "not" in W:
  print("Yes")
elif "that" in W:
  print("Yes")
elif "the" in W:
  print("Yes")
elif "you" in W:
  print("Yes")
else:
  print("No")